player_android.go

```diff
@@ -279,6 +292,19 @@ func (p *player) Write(data []uint8) (int, error) {
 }
 
 func (p *player) Close() error {
+	p.mu.Lock()
+	p.mu.Unlock()
```

Comment:
> It seems IllegalStateException is throwed if release method is called for the AudioTrack instance that is already released.

Hm? So `Close` callers should have responsibility of concurrency, and `Close` doesn't have to care about this, right?

> ah, sorry, I need to insert defer ...
  
Yes, thanks. But in the first place, I'm wondering if we really need to lock here.